### ISSUE TITLE
PhET confidentiality obligation MD file

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/phet-info#153]
In relation to PhET priviacy policy being drafted:

Create a PhET confidentiality obligation in phet-info that will apply to all PhET employees, agents, and contractors

PhET takes the confidentiality of user personal information very seriously. We employ reasonable security practices for protecting user information. We tightly restrict access to user information to PhET employees, agents, and contractors for which this access is essential; these parties are subject to confidentiality obligations and will be terminated if they fail to meet these obligations. Furthermore, user information is only accessed in cases requiring diagnostic, account service, and correctional purposes.

**Comment by ariel-phet** [comment]
@oliver-phet here is a first draft of a confidentiality policy 

https://github.com/phetsims/phet-info/blob/master/policies/Confidentiality-of-user-information.md

Would be good for both you and @kathy-phet to review

**Comment by oliver-phet** [comment]
Yep, I think this looks like it should cover our bases. @kathy-phet let me know when you have a chance to review.

**Comment by samreid** [comment]
I corrected some typos and spelling errors in the commit.

**Comment by jessegreenberg** [comment]
While reviewing read-for-review issues we found this one, @oliver-phet reassigning to you to move forward.

**Comment by oliver-phet** [comment]
I removed my assignment after completing review on Jan 28. This should just be assigned to @kathy-phet and included in the privacy policy project.

**Comment by oliver-phet** [comment]
We had everyone sign the obligation as part of the FY23 annual evaluations. Closing!
